perf8/plugins/_psutil.py

```diff
@@ -121,6 +121,9 @@ async def probe(self, pid):
 
         self.debug("Probing")
         probed_at = time.time()
+        if info.get("memory_info") is None:
```

Comment:
unclear why, but I was seeing instances where the test would fail and a bunch of the keys in the `info` object had `None` values. A few lines above, we return prematurely if fetching `info` hits any errors, so I figured it was safe to return early here if we didn't hit errors, but didn't fetch anything valuable.